src/app/cheat-sheets/game-base/material-processing/material-processing.component.html

```diff
@@ -23,6 +23,9 @@
             <th>
               <app-factorio-icon [icon]="dataService.getFactorioIcon('Express_transport_belt')"></app-factorio-icon>
             </th>
+            <th>
+              <app-factorio-icon [icon]="dataService.getFactorioIcon('Turbo_transport_belt')"></app-factorio-icon>
```

Comment:
Just fyi can add space age flag with
```suggestion
              <app-factorio-icon [icon]="dataService.getFactorioIcon('Turbo_transport_belt')"  [spaceAge]="true"></app-factorio-icon>
```
but i can take care of that